### ISSUE TITLE
Where are the C# communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/csharp#589]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is C# supported by one or more large organizations?
* Does C# have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about C#? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for C#? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in C#? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in C#?
* Are there popular newsletters for C#?
* Is C# taught at programming bootcamps? (If so, what are some examples?)
* Is C# taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about C#?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by ErikSchierboom** [comment]
> Is C# supported by one or more large organizations?

Yes, it is supported by Microsoft. Also, Samsung has been investing a lot in .NET Core, as their Tizen software runs on it. We used to have Xamarin as a medium-to-large organization supporting C#, but they have been acquisitioned by Microsoft.

> Does C# have an official community manager?

I don't know.

> Do you know of specific communities (online or offline) that are enthusiastic about C#? (Chat communities, forums, meetups, student clubs, etc)

Well, people on StackOverflow are often quite positive about C#. There are also the [.NET foundation forums](https://forums.dotnetfoundation.org/).

> Are there popular conferences for C#? (If so, what are some examples?)

There are the [NDC conferences](https://ndcconferences.com/), [Microsoft Build](https://www.microsoft.com/en-us/build), [dotnetConf](https://www.dotnetconf.net/), and many more. A nice overview can be found [here](https://blog.elmah.io/great-dot-net-conferences-to-attend/).

> Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in C#? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

There is the [Women in Tech community](https://partner.microsoft.com/en-US/community/wit).

> Are there specific groups or programs dedicated to mentoring people in C#?

Well, the [.NET Foundation](https://dotnetfoundation.org/) does sponsor Meetups and more.

> Are there popular newsletters for C#?

[dotNET Weekly](https://www.dotnetweekly.com/)

> Is C# taught at programming bootcamps? (If so, what are some examples?)

I don't know.

> Is C# taught at universities? (If so, what are some examples?)

I don't know.

**Comment by BillWagner** [comment]
> Is C# supported by one or more large organizations?

Microsoft, and it is an ECMA standard. The standards committee also has representatives from Google and HP.

> Does C# have an official community manager?
Not as a single person. The twitter account for the [.NET Foundation](https://twitter.com/dotnetfdn) is a good resource.

> Do you know of specific communities (online or offline) that are enthusiastic about C#? (Chat communities, forums, meetups, student clubs, etc)
Search for ".NET" on Meetup, and you'll find meetups in many physical communities.

> Are there popular conferences for C#? (If so, what are some examples?)
I'd add [VS Live](https://vslive.com/Home.aspx) and [DevIntersection](https://www.devintersection.com/)

> Are there popular newsletters for C#?
[The Code Project](https://www.codeproject.com/) is another one.

> Is C# taught at programming bootcamps? (If so, what are some examples?)
[Grand Circus](https://www.grandcircus.co/) is one. 

> Is C# taught at universities? (If so, what are some examples?)
Yes, but I'm not sure where. I'll find out.

**Comment by kytrinyx** [comment]
> C# taught at programming bootcamps?

I'm pretty sure https://sabio.la/ uses C#. I started googling and found this: https://www.thinkful.com/bootcamps/topic/c-sharp/ which might help.

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌸
